index.html

```diff
@@ -1831,322 +1831,291 @@ <h2>Proof</h2>
-the <code>@context</code> property. The
-<a href="https://www.w3.org/TR/json-ld/#the-context">JSON-LD Context</a>
-is described in detail in the [[!JSON-LD]] specification and its use is
-elaborated on in Section <a href="#extensibility"></a>.
-      </p>
+    <p class="issue" data-number="203">
+This requirement depends on the return of DID document metadata that still needs to be
+defined by this specification. Once defined, that should be linked from here.
+    </p>
 
-      <p>
-In general, the data model and syntaxes described in this document are designed
-such that developers can copy and paste examples into their software systems.
-The design goal of this approach is to provide a low barrier to entry while
-still ensuring global interoperability between a heterogeneous set of software
-systems. This section describes some of these approaches, which will likely go
-unnoticed by most developers, but whose details will be of interest to
-implementers. Noteworthy features provided by JSON-LD are:
-      </p>
+    <p>
+The production and consumption rules in this section apply to all implementations
+seeking to be fully compatible with independent implementations of the specification.
+Deployments of this specification MAY use a custom agreed-upon representation, including
+localized rules for handling properties not listed in the registry.
+See section ?? on Extensibility for more information.
+    </p>
 
-      <ul>
-        <li>
-The <code>@id</code> and <code>@type</code> keywords are aliased to
-<code>id</code> and <code>type</code> respectively, enabling developers to use
-this specification as idiomatic JSON.
-        </li>
-        <li>
-Data types, such as integers, dates, units of measure, and URLs, are
-automatically typed to provide stronger type guarantees for use cases that
-require them.
-        </li>
-        <li>
-The <code>@protected</code> properties feature of JSON-LD 1.1 is used to ensure
-that terms defined by this specification cannot be overridden. This means that
-as long as the same <code>@context</code> declaration is made at the top of a
-<a>DID document</a>, interoperability is guaranteed between implementations
-that use a JSON-LD processor and implementations that do not.
-        </li>
-      </ul>
+    <p class="issue" data-number="207">
+A link to a section on extensibility and conformance as it applies to data representations
+should be added here once that section has been written.
+    </p>
 
-      <section>
+    <section>
         <h2>
-  Contexts
+JSON
         </h2>
 
         <p>
-  When two software systems need to exchange data, they need to use terminology
-  and a protocol that both systems understand. The <code>@context</code>
-  property ensures that two systems operating on the same DID document are using
-  mutually agreed terminology.
+When producing and consuming DID documents that are in plain JSON (as noted by
+the document metadata), the following rules MUST be followed. 
```

Comment:
Hmm, again... do we trust the document metadata or the media type. The more variations there are, the more opportunity for misimplementation. Also, what happens when the media type disagrees with the document metadata? "application/json" media type for document containing "@context". The latter requires content sniffing, which is generally frowned upon. I suggest we specify that the media type is how you detect a plain JSON DID document from a JSON-LD document. We can also say that plain JSON DID documents MUST NOT contain a `@context` property (which I disagree with... suggest downgrading that to SHOULD NOT)... but again, the core issue here is how do you know something is a JSON document or a JSON-LD document? Content sniffing, media type, or both? If both, what are the rules when content sniffing disagrees with media type (typically, content sniffing wins... but again, let's try to avoid that anti-pattern).

Reply:
See above -- this is why we need document metadata, which includes the document media type. They will never be in conflict with each other because they aren't different things. 

The requirement to include the document type in this metadata information is exactly to avoid the kind of content sniffing you're talking about.

Reply:
> document metadata

I suggest that we stop using this term... it's confusing, because there is more than one type of document metadata. To me, media type is different from document metadata (the first being specific, the second being this amorphous term that means different things to different people).

Reply:
Media type is one field of document metadata. Until we have a more concrete term for the metadata mechanism, I suggest we use this one.

Reply:
We could just avoid using the "document metadata" term entirely because it looks like all of your language just requires the decision of what representation you're using to be triggered off of the media type (nothing else is taken into consideration), which I think is good. We don't need to talk about document metadata (because we don't really care how it's signaled to the consumer as long as it's signaled clearly).

I don't feel very strongly about this, just noting that words are being used w/o a clear definition... and we'll have to fix that later.

Reply:
I agree that it needs to be fixed later, but I think that it needs to be fixed by defining the metadata.